### ISSUE TITLE
"Site Mailbox" no longer exists - why is it the first suggestion?

### ISSUE THREAD

**Issue by kamereone** [MicrosoftDocs/OfficeDocs-Exchange#95]
Per Microsoft Support, the Site Mailbox feature was depreciated 15 months prior to this article's latest update.  Refer to https://support.office.com/en-us/article/Use-a-site-mailbox-to-collaborate-and-coordinate-team-email-57E361EB-39EF-400E-92DE-F6408DAB6B2B

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4bd8e7d2-4bf8-46e2-480f-9eca0c71cb65
* Version Independent ID: 60583132-eadf-5119-fc52-495cec0ba1f5
* Content: [Collaboration in Exchange Online](https://docs.microsoft.com/en-us/Exchange/collaboration-exo/collaboration-exo)
* Content Source: [Exchange/ExchangeOnline/collaboration-exo/collaboration-exo.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeOnline/collaboration-exo/collaboration-exo.md)
* Service: **exchange-online**
* Product: **exchange**
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by AndreaBarr** [comment]
@kamereone Thank you for this feedback

We appriciate you pointing this out. I will get the over to our SharePoint team so we can get this updated. 

I was able to get this into the backlog for the SharePoint team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
